core/trino-main/src/test/java/io/trino/execution/TestQueryManagerConfig.java

```diff
@@ -145,6 +146,9 @@ public void testExplicitPropertyMappings()
                 .put("fault-tolerant-execution-arbitrary-distribution-write-task-target-size-growth-factor", "2.5")
                 .put("fault-tolerant-execution-arbitrary-distribution-write-task-target-size-min", "6GB")
                 .put("fault-tolerant-execution-arbitrary-distribution-write-task-target-size-max", "10GB")
+                .put("fault-tolerant-execution-hash-distribution-compute-task-target-size", "1GB")
+                .put("fault-tolerant-execution-hash-distribution-write-task-target-size", "7GB")
+                .put("fault-tolerant-execution-hash-distribution-write-task-count-soft-upper-bound", "5000")
```

Comment:
maybe just fault-tolerant-execution-hash-distribution-write-max-task-count` or `fault-tolerant-execution-hash-distribution-target-task-count`?

Reply:
I will probably just do `target-max-count` and indicate it as a soft limit only.